django/forms/widgets.py

```diff
@@ -875,10 +875,10 @@ class SplitDateTimeWidget(MultiWidget):
     supports_microseconds = False
     template_name = 'django/forms/widgets/splitdatetime.html'
 
-    def __init__(self, attrs=None, date_format=None, time_format=None):
+    def __init__(self, attrs=None, date_format=None, time_format=None, date_attrs=None, time_attrs=None):
```

Comment:
You're welcome.

I see your point and it makes sense, although it feels rather magic, especially as values provided to `attrs` will override values in the others. This overriding behaviour should be probably be documented clearly on these widgets and on `MultiWidget` itself.

Documentation also needs to be replicated to `SplitHiddenDateTimeWidget` and release notes added.